pdc/apps/package/models.py

```diff
@@ -27,6 +27,7 @@ class RPM(models.Model):
     release             = models.CharField(max_length=200, db_index=True)
     arch                = models.CharField(max_length=200, db_index=True)  # nosrc
     srpm_name           = models.CharField(max_length=200, db_index=True)  # package (name of srpm)
+    koji_build_target   = models.CharField(max_length=200, null=True, blank=True, db_index=True)
```

Comment:
What if I want to use PDC without koji? How can I map builds to releases then?

Or how to deal with several koji instances?
For example Fedora uses primary koji and secondary-arch koji.
Then this field is useless, because PDC knows nothing about koji instances and this field refers to generic koji.

I'd rather create a relation and let someone/something outside PDC decide how builds map to releases.